app/src/main/java/com/humberto/tasky/agenda/presentation/agenda_details/components/AddEventAttendeeDialog.kt

```diff
@@ -37,7 +38,7 @@ fun AddEventAttendeeDialog(
     onAdd: () -> Unit,
     agendaItem: AgendaItemDetails.Event
 ) {
-    var isFocused by remember { mutableStateOf(false) }
+    var isFocused by rememberSaveable { mutableStateOf(false) }
```

Comment:
I'm not 100% sure myself here, but is focus in general always restored after a config change/process death? If it isn't, this might lead to a mini bug if you assume the text field is still focused even if it isn't anymore

Reply:
Actually I don't need to control if the field isFocused or not in this case since it's only one field right? Because when config change/process death causes recomposition, focusRequester will request focus to the field.
ps: I actually forgot to deal with process death but will do it :)

Reply:
I also noticed that when not manually requesting focus, isFocused with either remember or rememberSaveable loses its state on config changes.